assemble/bin/accumulo-cluster

```diff
@@ -465,6 +489,12 @@ gc:
 tserver:
   - localhost
 
+#sserver:
+#  - group:
+#    - default
+#  - default:
+#    - localhost
+#
```

Comment:
```suggestion
#sserver:
#  - default:
#    - localhost
#
```

Can we drop the group: section?

Reply:
No, I don't think so, unless you can figure it out. 

https://github.com/apache/accumulo/blob/main/core/src/main/java/org/apache/accumulo/core/conf/cluster/ClusterConfigParser.java#L45

Reply:
I suggested something in a comment, but not sure if it will work.